pipelines/samples/signing-operator/README.md

```diff
@@ -0,0 +1,174 @@
+# Sample image signing operator 
+The kabanero-security image signing operator is a sample operator for image signing which is capable to generate or import a keypair for image signing and create a secret resource which the image signing task consumes in the Kabanero pipeline.
+
+# About image signing operator
+
+The image signing opeator is for automating various configurations which are required enabling the image signing by the Tekton pipeline in Kanabero.
+The current code does following:
+- generate or import RSA keypair for the image signing based on the values of ImageSigning custom resource instance.
+- create the signature-secret-key secret resource from the RSA keypair. This secret is consued by the image signing task of Tekton pipeline.
+- When the ImageSignig custom resource is deleted, the corresponding signature-secret-key resource is also deleted.
+- If the generated signature-secret-key resource is deleted, the same secret will be created by the image signing operator.
+
+# Build and install the image signing operator
+
+## Prerequisite
+
+The image signing operator is developed using operator-sdk version 0.11.0.
+
+In order to build and deploy the operator, the following prerequisite packages need to be installed:
+- go version 1.13 or later
+- operator SDK version 1.11.0
+- Openshift Container Platform 4.2 CLI
+
+## Building the sample operator
+
+### Clone the image signing operator
+
+```
+git clone https://github.com/kabanero-io/kabanero-security/pipelines/samples/signing-operator
+cd signing-operator
+```
+
+You will need to examine the Makefile and set any necessary variables to push your container images to the correct repository. Especially, make sure that the variable of REPO is point to the corret repository, otherwise, the image will not pe pushed.
```

Comment:
Change "REPO is point to the corret" to "REPO is pointing to the correct"

Reply:
done.